mkdocs.1

```diff
@@ -0,0 +1,27 @@
+.TH "MKDOCS" "1" "0.1" "William Moreno Reyes" ""
+.SH "NAME"
+mkdocs \- Create HTML documentation from markdown sources
+.SH "SYNOPSIS"
+mkdocs [help|new|build|serve|gh\-deploy] {options}
+.SH "DESCRIPTION"
+MkDocs is a fast and simple way to create a website from source files written in Markdown, and configured with a YAML configuration file, the documentation  can be hosted anywhere, even in free hosting services like Read the Docs and GitHub Pages.
```

Comment:
Can we improve the wrapping of this? I see in your screenshot it doesn't look great. Maybe we need to manually wrap it?